transcripts/chaincode-labs/2019-06-12-ethan-heilman-network-partitioning-attacks.mdwn

```diff
@@ -0,0 +1,135 @@
+Ethan Heilman
+
+Bitcoin network partitioning & network-level privacy attacks
+
+Location: Chaincode Labs 2019 Residency
+
+Video: <https://www.youtube.com/watch?v=StnOVBbIpD8>
```

Comment:
Video: https://www.youtube.com/watch?v=StnOVBbIpD8

Reply:
the `<http://>` notation will cause ikiwiki to convert to a hyperlink

Reply:
Ah. Did not know that. Ok cool. Better to have `<http://>` then. I have never done that on any of mine.

Reply:
> I have never done that on any of mine.

Meh, it's fine, really doesn't matter too much.